Fix:
error: implicit declaration of function 'utf8len' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'mvwideaddstr' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'get_widech' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'check_script_file_with_current_encoding' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'fatal_error' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
https://git.savannah.gnu.org/cgit/gtypist.git/patch/?id=05639625b68131e648dfe2aec4dcc82ea6b95c6e
--- src/cursmenu.c.orig	2013-08-18 09:36:14.000000000 -0500
+++ src/cursmenu.c	2022-04-18 11:17:30.000000000 -0500
@@ -20,6 +20,7 @@
 #include "config.h"
 #include "cursmenu.h"
 #include "script.h"
+#include "utf8.h"
 
 #ifdef HAVE_PDCURSES
 #include <curses.h>
--- src/gtypist.c.orig	2014-08-12 01:29:13.000000000 -0500
+++ src/gtypist.c	2022-04-18 11:17:30.000000000 -0500
@@ -51,6 +51,7 @@
 #include "error.h"
 #include "gtypist.h"
 #include "utf8.h"
+#include "script.h"
 
 #include "gettext.h"
 #define _(String) gettext (String)
--- src/script.h.orig	2013-08-18 09:36:14.000000000 -0500
+++ src/script.h	2022-04-18 11:17:30.000000000 -0500
@@ -89,6 +89,7 @@
 extern void seek_label( FILE *script, char *label, char *ref_line );
 extern int hash_label( char *label );
 extern void do_exit( FILE *script );
+void check_script_file_with_current_encoding( FILE *script );
 
 
 extern void bind_F12 (const char *);	// Defined in gtypist.c
--- src/utf8.c.orig	2013-08-18 09:36:14.000000000 -0500
+++ src/utf8.c	2022-04-18 11:17:30.000000000 -0500
@@ -19,6 +19,7 @@
 
 #include "config.h"
 #include "utf8.h"
+#include "error.h"
 
 #ifdef HAVE_PDCURSES
 #include <curses.h>
